api/prisma/migrations/18_single_use_code_expiration_translations/migration.sql

```diff
@@ -0,0 +1,5 @@
+-- Update single-use code expiration translations
+UPDATE translations
```

Comment:
Curious why we went the migration route vs the script runner?

Reply:
Because I forgot about the script runner 😞 I'll work on that change now!

Reply:
Updated with new approach and testing instructions if you're curious 🐦‍⬛